src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -16167,7 +16189,9 @@ instruct CallLeafNoFPDirect(method meth)
 // Also known as an 'interprocedural jump'.
 // Target of jump will eventually return to caller.
 // TailJump below removes the return address.
-instruct TailCalljmpInd(iRegPNoSp jump_target, inline_cache_RegP method_ptr)
+// Don't use rfp for the jump_target because the
+// MachEpilogNode that is inserted above will kill it.
```

Comment:
This is a somewhat opaque comment. Please say that a `MachEpilogNode` has already been emitted, so the current rfp contains a value that belongs to the caller of this method, so it must not be used.

Reply:
Thanks Andrew. I adjusted the comments, please let me know what you think.